native/cocos/renderer/gfx-metal/MTLDevice.h

```diff
@@ -86,6 +86,10 @@ class CCMTLDevice final : public Device {
             _swapchains.erase(iter);
         }
     }
+    
+    CommandBuffer* transferCommandBuffer() const;
+    
+    void writeBuffer(Buffer* buffer, const void* data, size_t size);
```

Comment:
Why not use uint32_t?

Reply:
@hana-alice

Reply:
webgpu decorate buffer size with uint64_t, so try change buffer size to try keep consistent here . 
if size_t is not needed: https://github.com/star-e/cocos-engine/pull/96

Reply:
I think uint32_t is enough as our engine uses uint32_t for other codes.

Reply:
pr has been merged. Now metal uses uint32_t